analyzers/src/SonarAnalyzer.Common/Extensions/SyntaxTreeExtensions.cs

```diff
@@ -38,7 +38,7 @@ public static bool IsGenerated(this SyntaxTree tree, GeneratedCodeRecognizer gen
             return false;
         }
         var cache = GeneratedCodeCache.GetOrCreateValue(compilation);
-        // Hotpath: Don't use cache.GetOrAdd that takes a factory method. It allocates a delegate which causes GC preasure.
+        // Hot path: Don't use cache.GetOrAdd that takes a factory method. It allocates a delegate which causes GC preasure.
```

Comment:
Was supposed to be `pressure`, instead of `preasure`?

Reply:
I've noticed that typo a bit later and fixed it in: https://github.com/SonarSource/sonar-dotnet/pull/7916/files